### PR TITLE
[ASCN-375] Setup Serilog and Kubernetes console logging

### PR DESCRIPTION
**Pull request by WouterDeKort**
This PR is for [ASCN-375](https://stackoverflow.atlassian.net/browse/ASCN-375).

This PR adds Serilog and configures the app so exceptions that go to Exceptional are also logged to the console. This allows us to see the errors in DataDog.

[ASCN-375]: https://stackoverflow.atlassian.net/browse/ASCN-375?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/Opserver.Web/Startup.cs [hunk 1]
```diff
@@ -65,6 +68,7 @@ public void ConfigureServices(IServiceCollection services)
                     {
                         settings.UseExceptionalPageOnThrow = true;
```
Comment (juanrodriguezcen) [inline src/Opserver.Web/Startup.cs:69]: This is the problem, when exceptional has `UseExceptionalPageOnThrow` it captures the exception, renders the page and stops the response pipeline there, so it won't reach to your `UseExceptionHandler`. If you check out the monolith, we do:

```c#
settings.UseExceptionalPageOnThrow = Env.Tier == DeploymentTier.Local;
```

This means that unhandled exceptions in local get the Exceptional page but are not logged into console, while in dev/prod it is logged to console.

Comment (WouterDeKort) [reply src/Opserver.Web/Startup.cs:69]: I'll add this!

However, in my scenario the exception is not unhandled. The exception is caught, send to Exceptional and then the `ExceptionalAfterLog` event handler is triggered which is hit and does run `Serilog.Log`. But unfortunately, it does not show up in my Visual Studio debug output or in the Docker container logs.

**Review by juanrodriguezcen — APPROVED** [review]
lgtm

**Review by stephen-vakil — APPROVED** [review]
tested/verified locally

**Pull request merged**